internal/initwd/copy_dir.go

```diff
@@ -59,6 +60,27 @@ func copyDir(dst, src string) error {
 			return nil
 		}
 
+		// info.IsDir() returns false for symlinks, even if the symlink is a
+		// directory.
+		// TODO: This may be fixed - or different - in go1.12, at which point we
```

Comment:
This shouldn't change in 1.12. This is the expected behavior with `stat`, which is why there is also `lstat`. The `fileinfo.Walk` function is documented as using the former.

Reply:
thanks! I removed the incorrect comment.